pb_nautobot_install.yml

```diff
@@ -87,14 +79,22 @@
         priv: "ALL"
         state: "present"
 
-    - name: "INSTALL NAUTOBOT"
+    - name: "INSTALL NAUTOBOT AND PLUGINS"
       ansible.builtin.pip:
-        name: "nautobot=={{ lookup('env', 'NAUTOBOT_VERSION') }}"
+        name: "{{ nautobot_item }}"
         virtualenv: "{{ nautobot_root }}"
         virtualenv_command: "python3 -m venv"
-
-    - name: "RUN NAUTOBOT-SERVER INIT"
-      ansible.builtin.command: "{{ nautobot_root }}/bin/nautobot-server init"
+      loop:
+        - "nautobot=={{ lookup('env', 'NAUTOBOT_VERSION') }}"
```

Comment:
You might need to watch that the lines below don't upgrade/downgrade the installed Nautobot version. Reason to not use a `requirements.txt` file to ensure that Pip sees the whole picture of the desired installed? Otherwise, I would consider using the `--constraints` pattern of Pip. We needed this to not have surprise version changes.

Reply:
https://github.com/nautobot/nautobot-plugin-chatops/blob/e9456b2cb6193dadace3b6bd14254fc76321fbb6/development/Dockerfile#L56-L76